Test/Tests/RealtimeClientConnectionTests.swift

```diff
@@ -4139,16 +4141,28 @@ class RealtimeClientConnectionTests: XCTestCase {
         }
 
         XCTAssertEqual(urlConnections.count, 3)
-        XCTAssertTrue(NSRegularExpression.match(urlConnections.at(0)?.absoluteString, pattern: "//realtime.ably.io"))
-        XCTAssertTrue(NSRegularExpression.match(urlConnections.at(1)?.absoluteString, pattern: "//[a-e].ably-realtime.com"))
```

Comment:
So we're no longer testing that the fallback URLs are correctly formed?

Reply:
Yeah, I missed that lines when came up with a proper check somewhere else - https://github.com/ably/ably-cocoa/pull/2017/commits/2a9b845dad248acaa9cfc2584bd3a52481749f63